src/train/isolator/isolation_trainers/mlp.py

```diff
@@ -0,0 +1,101 @@
+from isolation_trainer import IsolationTrainer
+import keras
+from keras import layers
+from tensorflow.keras.constraints import non_neg as nonneg
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import MinMaxScaler
+import numpy as np
+
+def mlp_1(input_dim):
+    model = keras.Sequential([
+    layers.Dense(400, activation='relu', input_shape=[input_dim], kernel_constraint=nonneg()),
+    layers.Dense(1)
+    ])
+    model.compile(loss='mse',
+                optimizer='adam',
+                metrics=['mae', 'mse'])
+    return model
+
+def mlp_2(input_dim):
+    model = keras.Sequential([
+    layers.Dense(400, activation='relu', input_shape=[input_dim], kernel_constraint=nonneg()),
+    layers.Dense(250, activation='relu', input_shape=[input_dim], kernel_constraint=nonneg()),
+    layers.Dense(1)
+    ])
+    model.compile(loss='mse',
+                optimizer='adam',
+                metrics=['mae', 'mse'])
+    return model
+
+def mlp_3(input_dim):
+    model = keras.Sequential([
+    layers.Dense(400, activation='relu', input_shape=[input_dim], kernel_constraint=nonneg()),
+    layers.Dense(250, activation='relu', input_shape=[input_dim], kernel_constraint=nonneg()),
+    layers.Dense(150, activation='relu', input_shape=[input_dim], kernel_constraint=nonneg()),
+    layers.Dense(1)
+    ])
+    model.compile(loss='mse',
+                optimizer='adam',
+                metrics=['mae', 'mse'])
+    return model
+
+def mlp_2_do(input_dim):
+    model = keras.Sequential([
```

Comment:
Looks like a bit of repetetive code? I am pretty sure tensorflow keras with functional api might be better.

Reply:
Oh... I need to remove it. it is draft code. Thanks.